src/applications/vaos/new-appointment/components/ConfirmationPage/ConfirmationDirectScheduleInfoV2.jsx

```diff
@@ -14,6 +14,7 @@ import {
   getTimezoneByFacilityId,
 } from '../../../utils/timezone';
 import { GA_PREFIX, PURPOSE_TEXT } from '../../../utils/constants';
+import { getTypeOfCareById } from '../../../services/appointment/transformers.v2';
```

Comment:
Since this file is only used by the Var Resources api (v0), use should call on the v0 transformer
```suggestion
import { getTypeOfCareById } from '../../../services/appointment/transformers';
```